src/scripts/docs/FormApp.jsx

```diff
@@ -187,6 +188,17 @@ const FormApp = () => (
           <Radio value="3" label="Radio Three" />
         </RadioGroup>
         <RhinoSwitch label="Switcher" name="exampleSwitch1" />
+        <SlidingRadio
+          selectedValue="2"
+          label="Sliding Radios"
+          name="exampleSlidingRadio1"
+          slidingRadioContainerClass="rhinoslidingradio__container"
+          options={[
+            { label: 'Denied', value: '1' },
```

Comment:
Here is where I think we can pass down the color class for each option as part of each option object.

Reply:
👍